extensions/amp-consent/0.1/consent-info.js

```diff
@@ -286,3 +326,54 @@ export function getConsentStateValue(enumState) {
 
   return 'unknown';
 }
+
+/**
+ * Converts metadata to stroage value:
+ * {'gdprApplies': true, 'consentStringType': 'tcf-v2'} =>
+ * {'ga': true, 'cst': 'tcf-v2'}
+ *
+ * @param {Object} unused
+ * @return {Object}
+ */
+function composeMetadataStoreValue(unused) {
+  const storageMetadata = {};
+  // TODO(micajuineho) if (metadata['gdprApplies']) {...}
```

Comment:
IMO: `composeMetadataStoreValue` and `convertStorageMetadata` are a pair of helper methods. What about we implement them together in a separate PR?

Reply:
That was my idea as well. Just wanted to lay the skeleton out here, and then add implementation as we add `gdprApplies` and `consentStringType` to metadata. Is that ok?

Reply:
Sounds good to me! Correct me, but I think you have `convertStorageMetadata()` implemented?

Reply:
Yea, it shouldn't do anything, and I wanted to make sure my ideas were valid by fleshing it out. Should I remove it?

Reply:
SG. Thanks